content/zh-cn/dogepedia/articles/history-of-dogecoin.md

```diff
@@ -1,28 +1,26 @@
 +++
-title = "The History of Dogecoin"
+title = "狗狗币的历史"
 date = "2021-10-22"
 type = "article"
 
 [ author ]
   name = "Dogecoin"
 +++
- 
-Dogecoin was originally created as a joke in late 2013 by Billy Markus and Jackson Palmer. Both Billy and Jackson thought that cryptocurrency was being taken far too seriously and wasn’t much fun. Jackson created the Dogecoin.com website and would have been, for a while, the public face of Dogecoin, while Billy worked as a solo developer on the first 4 releases.
 
-![image info](/assets/images/dogepedia/12.png)
-
-On December 6th, 2013, Dogecoin was launched taking influence from the famous Doge meme: an image of a very adorable Shiba Inu dog with multi-coloured comic sans text that appears as the dog’s “thought bubbles”. 
+狗狗币是2013年底作为一个玩笑由Billy Markus和Jackson Palmer创建的。Billy和Jackson觉得加密货币太严肃了，没有什么好玩的。于是，Jackson创建了Dogecoin.com网站，这个网站在很长时间都是狗狗币的官方门户。然后，Billy独自开发了狗狗币的前4个版本。
 
+![image info](/assets/images/dogepedia/12.png)
 
-This famous meme was based on a 2010 photograph of a Japanese female Shiba Inu dog called **Kabosu**, who was adopted by Japanese kindergarten teacher Atsuko Satō from an animal shelter in 2008.  
+2013年12月6日，在当时风靡世界的Doge梗文化的影响，狗狗币诞生了。它是一张非常可爱的没有文字的漫画柴犬形象，看起来充满了狗狗的思想智慧。
 
-Much to Billy and Jackson’s surprise, Dogecoin was a near-instant success, and gained tremendous popularity on sites such as Reddit, where it was —and still is — used as a tipping currency. When Dogecoin was just 2-weeks old, it reached more transactions per day than Bitcoin. Moreover, within the first month there were over one-million unique visitors to this website!  
+这个著名的梗文化发源于2010年网上出现的一只日本雌性柴犬**Kabosu**的照片，这只狗狗是日本幼儿园老师Atsuko Satō在2008年从动物收容所收养的。
```

Comment:
梗文化->模因文化

Reply:
Likewise with this one. This is demonstrates a real life example of how Doge is every bit as capable of monetizing social media in a way individuals can retain ownership and profitability rather than whatever leftovers Google decides to share with creators. Great use case for adoption